packages/flutter/lib/src/material/back_button.dart

```diff
@@ -9,6 +9,33 @@ import 'icon_button.dart';
 import 'icons.dart';
 import 'theme.dart';
 
+/// A "back" icon that's appropriate for the current [TargetPlatform].
+///
+/// See also:
+///
+///  * [BackButton], an [IconButton] with a [BackButtonIcon] that
+///  * [IconButton], which is a more general widget for creating buttons with
+///    icons.
```

Comment:
also maybe those should link back here, though that might be more complicated since i think that file is generated from a script